app/assets/javascripts/locationForm.js

```diff
@@ -58,6 +58,13 @@ function addGrid(map)
    _renderCells: function(bounds) {
      var cells = this._cellsInBounds(bounds);
      this.focused_cell_id = cells.length == 0 ? null : cells[0].id;
+     window.gridSquareNWCorner = function(){
```

Comment:
Hmm, does this have to be attached to the window? That basically makes it a global. Where is it called from?